### PR TITLE
CATC8: added test config change triggers convergence

### LINKED ISSUE #1317: CATC8: Update launch config after OOB delete
aka - trigger convergence after OOB delete
OOBD: Update the launch config after an OOB delete
	- Create a group and scale to N, where N > min
	- Delete a server out of band
	- Update the ~~launch~~ group config (ex. change the group MAX)
	- Validate that updating the config triggered convergence back to N servers
		- White-box test?

### PR DESCRIPTION
**Pull request by kivattik**
Fixes #1317

### CODE REVIEW

File: otter/integration/tests/test_convergence.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def test_reaction_to_oob_deletion_then_scale_up(self):
             )
         )
 
-    def test_reaction_to_oob_server_deletion(self):
+    def test_reaction_to_oob_server_deletion_below_min(self):
```
Comment (cyli) [inline otter/integration/tests/test_convergence.py]: +1 thanks for the test name change to be more clear :)

**Comment by sam-falvo** [comment]
+1 if it passes Travis.

**Comment by cyli** [comment]
LGTM :)

**Comment by cyli** [comment]
Again, coveralls.  :|  Just merging.

**Pull request merged**